deployments/helm/hephaestus/values.yaml

```diff
@@ -88,6 +88,9 @@ controller:
   # Defaults to "10m"
   poolMaxIdleTime: null
 
+  # Secrets (name: path) to expose into builds that request it
+  secrets: {}
```

Comment:
Yeah, I struggle with this since there's going to be some coupling either way:

* The thing invoking the build has to somehow know what's attached to the hephaestus pod, but that invoker can determine the ID of the secret and pass that into the Dockerfile in a single place.
* Here, we at least keep the mounting of the configmap and reference to it in the same place, but the Dockerfile just uses a "magic" ID that comes from seemingly nowhere.

Reply:
That is a bit of a pickle, isn't it? I think this will suffice for the first cut and we can figure the final solution in the near future like we discussed offline today.